digital-deck-server/services/utilities.js

```diff
@@ -0,0 +1,44 @@
+let concSessions = [];
+
+/**
+ * Returns all the sessions
+ * 
+ * @return [array of {json} ] The array of json containing information of all sessions
+ */
+function getConcSessions()
+{
+    return concSessions;
+}
+
+/**
+ * Adds a new session in concurrent sessions
+ * 
+ * @param session The session to add
+ */
+function addSession(session)
+{
+    concSessions.push(session);
+}
+
+/**
+ * Returns a session based on id
+ * 
+ * @param id The session to add
+ */
+function getSession(id)
+{
+    return concSessions.find(session => session.sessionId === id);
+}
```

Comment:
I would suggest to move this functionality to ```playCardsService.js``` to keep things consistent and for future player to player plays

Reply:
The reason it is not in `session.js` is because after db is introduced the models folder will only have schemas and shouldn't have  any methods in it